airbyte-webapp/package.json

```diff
@@ -16,10 +16,11 @@
     "format": "prettier --write 'src/**/*.{ts,tsx}'",
     "storybook": "start-storybook -p 9009 --quiet",
     "build:storybook": "build-storybook -o 'build/storybook'",
-    "lint": "eslint --ext js,ts,tsx src",
+    "lint": "eslint --ext js,ts,tsx src && stylelint \"**/*.{css,scss}\"",
```

Comment:
I think some people without a permanent web setup that sometimes change the FE code, will use `npm run lint -- --fix` at the moment to apply fixes. With merging the stylelint into the eslint task here, that would not work anymore I think? Maybe it would make sense to just leave those tasks separate.

Reply:
Done!